scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -182,15 +182,11 @@ _uninstall_user_mode_ubuntu()
            "intel-igc-opencl")
 
     for package in "${PACKAGES[@]}"; do
-        found_package=$(dpkg-query -W -f='${binary:Package}\n' ${package})
-        if [[ $? -eq 0 ]]; then
-            echo Found $found_package installed, uninstalling...
-            dpkg --purge $found_package
-            if [[ $? -ne 0 ]]; then
-                echo "ERROR: unable to remove $found_package" >&2
-                echo "       please resolve it manually and try to launch the script again." >&2
-                exit $EXIT_FAILURE
-            fi
+        apt remove -y $package
+        if [[ $? -ne 0 ]]; then
```

Comment:
The absent package may cause an error, but this should be a valid case. We still have to check if the package installed on the system before removing it.